.env.example

```
@@ -44,6 +44,12 @@ EXPO_PUBLIC_POSTHOG_KEY=
 BETTER_AUTH_SECRET=
 NEXT_PUBLIC_COOKIE_DOMAIN=
 
+# Sign in with Apple. Native mobile sign-in only needs the bundle id defaults;
+# set a Services ID + generated client secret if the web flow is ever enabled.
+APPLE_CLIENT_ID=
+APPLE_CLIENT_SECRET=
+APPLE_APP_BUNDLE_IDENTIFIER=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APPLE_APP_BUNDLE_IDENTIFIER key should go before the APPLE_CLIENT_ID key

(UnorderedKey)